src/main/java/core/basesyntax/service/impl/ReporterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReporterService;
+import java.util.Map;
+import java.util.Set;
+
+public class ReporterServiceImpl implements ReporterService {
+    private static final String HEADER_TITLE = "fruit,quantity";
+    private static final String COMA = ",";
+
+    @Override
+    public String createReport(Set<Map.Entry<String, Integer>> data) {
+        StringBuilder builder = new StringBuilder(HEADER_TITLE);
+        for (Map.Entry<String, Integer> fruit : data) {
+            builder.append(System.lineSeparator())
+                    .append(fruit.getKey() + COMA + fruit.getValue());
```

Comment:
use `StringBuilder` to append

Reply:
Do you mean that I need to rename my `builder `to `stringBuilder`?